frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/getting-started-section.tsx

```diff
@@ -1,31 +1,39 @@
 import * as React from 'react';
 
-import { FLAGS } from '@console/shared';
+import {
+  FLAGS,
+  GETTING_STARTED_USER_SETTINGS_KEY_CLUSTER_DASHBOARD,
+  useUserSettings,
+} from '@console/shared';
 import { useFlag } from '@console/shared/src/hooks/flag';
 import {
-  GettingStartedGrid,
-  useGettingStartedShowState,
-  GettingStartedShowState,
+  GettingStartedExpandableGrid,
   QuickStartGettingStartedCard,
 } from '@console/shared/src/components/getting-started';
 
-import { USER_SETTINGS_KEY } from './constants';
 import { ClusterSetupGettingStartedCard } from './cluster-setup-getting-started-card';
 import { ExploreAdminFeaturesGettingStartedCard } from './explore-admin-features-getting-started-card';
 
 import './getting-started-section.scss';
 
 export const GettingStartedSection: React.FC = () => {
   const openshiftFlag = useFlag(FLAGS.OPENSHIFT);
-  const [showState, setShowState, showStateLoaded] = useGettingStartedShowState(USER_SETTINGS_KEY);
 
-  if (!openshiftFlag || !showStateLoaded || showState !== GettingStartedShowState.SHOW) {
+  const [isGettingStartedSectionOpen, setIsGettingStartedSectionOpen] = useUserSettings<boolean>(
+    GETTING_STARTED_USER_SETTINGS_KEY_CLUSTER_DASHBOARD,
+    true,
+  );
+
+  if (!openshiftFlag) {
     return null;
   }
 
   return (
     <div className="co-dashboard-getting-started-section">
-      <GettingStartedGrid onHide={() => setShowState(GettingStartedShowState.HIDE)}>
+      <GettingStartedExpandableGrid
+        isGettingStartedSectionOpen={isGettingStartedSectionOpen}
+        setIsGettingStartedSectionOpen={setIsGettingStartedSectionOpen}
+      >
```

Comment:
These prop names are repetitive. Isn't `isOpen` and `onToggle`/`setIsOpen` enough here?

Reply:
Updated the PR for this change